email_mailbox/src/components/RestoreBackupProgressPopup.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import string from './../lang';
+import './restorebackupprogresspopup.scss';
+
+const { header, cancelLabel } = string.popups.restore_backup_progress;
+
+const ManualSyncDeviceApprovedPopup = props => (
```

Comment:
Why is this function called like this when the filename is `RestoreBackupProgressPopup`